code/components/citizen-resources-gta/src/GameEventFunctions.cpp

```diff
@@ -9,6 +9,44 @@
 
 static InitFunction initFunction([]
 {
+	OnTriggerGameEventReact.Connect([](std::string_view data)
+	{
+		auto resman = Instance<fx::ResourceManager>::Get();
+		auto rec = resman->GetComponent<fx::ResourceEventManagerComponent>();
+
+		/*NETEV gameEventReact CLIENT
+		/#*
+		 * An event that is triggered when the game triggers an internal network event.
+		 *
+		 * @param name - The name of the triggered event.
+		 * @param id - The id of the triggered event.
+		 * @param entity - The entity reacting.
+		 * @param data - The type-specific event data.
+		 #/
+		declare function gameEventReact(name: string, id: number, entity: number, data: var[]): void;
+		*/
+		rec->QueueEvent("gameEventReact", std::string(data), "");
```

Comment:
As with #690, these events should ideally be scoped (i.e. the vtable/event name being present in the event name) to minimize HLL perf impact.

Reply:
Interesting idea, I'll append it.